test/cases_all.sh

```diff
@@ -2262,10 +2248,7 @@ test_log_option() {
   assertEquals 1 "$(grep -ac 'GEGE_BBBB_' < "${_log_file}")"
 
   close_tmux_session "$_socket_file"
-  rm -f "${_tmpdir}"/logs/*
-  rmdir "${_tmpdir}"/logs
-  rm -f "${_tmpdir}"/fin/*
-  rmdir "${_tmpdir}"/fin
+  rm -rf "${_logdir}" "${_tmpdir}"/fin
```

Comment:
Do not delete `"${_logdir}"` since it is not being used this part.

```suggestion
  rm -rf "${_tmpdir}"/{fin,logs}
```